### ISSUE TITLE
Unexpected error "Skipping non-existent file"

### ISSUE THREAD

**Issue by Southclaws** [IBM/openapi-validator#516]
Not really sure what else to say 😅 

![image](https://user-images.githubusercontent.com/1636971/202756387-58d13512-410e-4ec8-a28a-6924f8837a83.png)

Perhaps there's another problem and the error message is wrong?

**Comment by dpopp07** [comment]
Hmm it looks like you're running on Windows - is that correct? This issue has popped up before, the backslash confuses the code even though it should treat file separators generically

**Comment by Southclaws** [comment]
Yeah, Windows. Figured so, Windows accepts either slash so `/` works but native shells on Windows like PowerShell and Nushell tab-complete to the OS default of `\`.

Node.js has APIs for each operating system so you can switch the parser based on `path.sep`. Unfortunately it defaults to POSIX only but you can call `path.win32` on Windows and `path.posix` on Linux.
